samples/subsys/power/stm32_pm/lptim/src/main.c

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2016 Intel Corporation.
+ * Copyright (c) 2019 STMicroelectronics.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "sample.h"
+
+struct device *gpio_port, *gpio_port0;
+
+/* 1000 msec = 1 sec */
+
+#define WAIT_TIME 100
```

Comment:
Let's set to 500ms to make it more the sample result more visible

Reply:
changed